### PR TITLE
*: Enable distributed count with TiKV

### PR DESCRIPTION
**Pull request by shenli**
@coocood @disksing PTAL
Merge after https://github.com/pingcap/tikv/pull/679

### CODE REVIEW

File: store/tikv/coprocessor.go [hunk 1]
```diff
@@ -37,7 +37,12 @@ type CopClient struct {
 func (c *CopClient) SupportRequestType(reqType, subType int64) bool {
 	switch reqType {
 	case kv.ReqTypeSelect:
-		return supportExpr(tipb.ExprType(subType))
+		switch subType {
+		case kv.ReqSubTypeAgg:
```
Comment (disksing) [inline store/tikv/coprocessor.go]: Suggest to remove `ReqSubTypeAgg` and use corresponding `exprType` directly to check.

**Comment by shenli** [comment]
@disksing PTAL

**Comment by disksing** [comment]
LGTM.

**Comment by coocood** [comment]
LGTM

**Pull request merged**